build.sbt

```diff
@@ -579,17 +595,34 @@ lazy val `baas-smoke-tests` = project.in(file("baas-smoke-tests"))
   .settings(noPublishSettings)
   .settings(
     moduleName := "baas-smoke-tests",
+    commands += buildExampleDockerCommand,
     libraryDependencies ++= Seq() ++
       testDeps(
         http4sDsl,
         http4sClient,
         circe,
         slf4jApi,
         slf4jSimple,
-        logback,
         scalaTest,
         scalaCheck
       )
   )
-  .dependsOn(`baas-node-client`, `baas-client-example`)
+  .dependsOn(
+    `baas-node-client`,
+    `baas-client-example`,
+    `baas-interaction-example-make-payment`,
+    `baas-interaction-example-reserve-items`,
+    `baas-interaction-example-ship-items`)
 
+lazy val `sbt-baas-docker-generate` = project.in(file("sbt-baas-docker-generate"))
```

Comment:
perhaps worth adding a comment like
"// workaround to let plugin be used in the same project"